_scripts/syncserver.sh

```diff
@@ -3,6 +3,8 @@
 docker run --rm --name syncserver \
   -p 5000:5000 \
   -e SYNCSERVER_PUBLIC_URL=http://localhost:5000 \
+  -e SYNCSERVER_IDENTITY_PROVIDER=http://host.docker.internal:3030 \
+  -e SYNCSERVER_BROWSERID_VERIFIER=http://host.docker.internal:5050 \
```

Comment:
This has to make outgoing calls to the locally-running verifier and auth-server, but `--net=host` doesn't work in docker for windows.  The above works for me, but will need testing on other folks machines as well.

Reply:
I was trying to get this to work locally and had different values that obviously didn't work!